adobe_vipm/adobe/errors.py

```diff
@@ -39,9 +40,9 @@ def __repr__(self) -> str:
         return str(self.payload)
 
 
-def wrap_http_error(func: Callable[..., T]):
+def wrap_http_error(func: Callable[Param, RetType]) -> Callable[Param, RetType]:
     @wraps(func)
-    def _wrapper(*args, **kwargs) -> T:
+    def _wrapper(*args: Param.args, **kwargs: Param.kwargs) -> RetType:
```

Comment:
Decided not to comment it on previous PRs, but you did it by yourself now. Great!